plc4j/drivers/modbus/src/test/java/org/apache/plc4x/java/modbus/ManualModbusTCPDriverTest.java

```diff
@@ -69,22 +92,22 @@ public ManualModbusTCPDriverTest(String connectionString) {
     }
 
     public static void main(String[] args) throws Exception {
-        ManualModbusTCPDriverTest test = new ManualModbusTCPDriverTest("modbus-tcp://192.168.23.30");
+        ManualModbusTCPDriverTest test = new ManualModbusTCPDriverTest("modbus-tcp://10.10.1.200:10502?default-payload-byte-order=LITTLE_ENDIAN_BYTE_SWAP");
         test.addTestCase("holding-register:1:BOOL", new PlcBOOL(true)); // 0001
         test.addTestCase("holding-register:2:BYTE", new PlcBYTE(42)); // 2A
         //test.addTestCase("holding-register:3:WORD", new PlcWORD(42424)); // A5B8
-        test.addTestCase("holding-register:4:DWORD", new PlcDWORD(4242442424L)); // FCDE 88B8
+        test.addTestCase("holding-register:4:DWORD", new PlcDWORD(424244242L)); // 1949 7412
```

Comment:
Why change this to the smaller number? 4242442424 is a 32 bit unsigned value. Adding one more digit would make it a 33 bit number ... if you needed to go down to a 29-bit number, then I think there might be something wrong with the decoding.

Reply:
This is because the application used (MonbusTools), his field editor does not allow me to enter the last digit. It does not affect the result since what is important is the order of the bytes.

Reply:
Yeah ... ok ... at least that's a reason I would accept ... possibly entering it as a signed integer would work? so probably some large negative number? (-52524872) then we wouldn't be adjusting our tests to the bugs of a tool we use and would actually also test the boundaries of signed and unsigned integers.